vpn-policy-routing/files/vpn-policy-routing.init

```diff
@@ -656,9 +656,16 @@ table_create(){
 			if [ -z "$gw6" ] || [ "$gw6" = "::/0" ]; then
 				ip -6 route add unreachable default table "$tid" || ipv6_error=1
 			else
-				ip -6 route list table main | grep " dev $dev6 " | while read -r i; do
-					ip -6 route add $i table "$tid" >/dev/null 2>&1 || ipv6_error=1
-				done
+				if ip -6 route list table main | grep " dev $dev6 " >/dev/null 2>&1; then
+					output 2 "IPv6 routes found for '$dev6'. Making a copy."
+					ip -6 route list table main | grep " dev $dev6 " | while read -r i; do
+						ip -6 route add $i table "$tid" >/dev/null 2>&1 || ipv6_error=1
+					done
+				else 
+					output 2 "No IPv6 routes found. Adding a default route based on '$dev6' IP address." 
+					ip -6 route add $(ip -6 -o a show $dev6 | awk '{print $4}') dev $dev6 table $tid >/dev/null 2>&1 || ipv6_error=1
```

Comment:
Please do shellcheck/double-quotes. Please confirm wherever this has been tested with the default OpenWrt ip implementation (still busybox?), `ip` package and `ip-full` package. I remember there were differences in parameters they accept.